### DISCUSSION TITLE
Create a CSS variable for error/invalid conditions

### DISCUSSION THREAD

**Discussion by psarno** [primefaces/community#1385] [PrimeNG]
It would be extremely helpful if in your themes you exposed a CSS variable representing an appropriate error color for that theme.

You already have a `p-error` class, but there are scenarios where you can't use a class and need access to the underlying color value.  This becomes a problem when you can dynamically switch themes.

As an example, right now I need to apply a class to a table row.  I can't use `p-error` because it's less specific than the table row's `color` in the theme, so it gets overriden.   If I had access to the color via a CSS variable, this would be solvable.

It would also be beneficial for you, because you duplicate the error state a lot in the theme CSS files, such as for invalid states and other areas.

If you just have:

`--error-state: [error color for theme]`

Then you can set:

`.p-error { color: var(--error-state); }`

And I could get access to `--error-state`.  Win/win, no?

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the valuable feedback, we're working on a new design system.
